### PR TITLE
SAK-49649 SakaiGrader student's HTML file attachments are inaccessible to the grader

### PR DESCRIPTION
**Pull request by st-manu**

### CODE REVIEW

**Comment by adrianfish** [comment]
@st-manu I've pushed a commit onto your branch where I:

Update the kernel method getHtmlForRef to handle the html content type.
removed your newly added lit block and just added a filter for the string "InlineSub" in the attachments loop.

File: webcomponents/tool/src/main/frontend/packages/sakai-grader/src/SakaiGrader.js [hunk 1]
```diff
@@ -333,21 +333,17 @@ export class SakaiGrader extends graderRenderingMixin(gradableDataMixin(SakaiEle
...
     const type = this._selectedAttachment.type;
 
-    if (type === "text/html") {
-      this._submittedTextMode = true;
-    } else {
-      this._submittedTextMode = false;
-      this.previewMode = true;
-      let preview = this._submission.previewableAttachments[this._selectedAttachment.ref];
-      preview = !preview && (type.startsWith("image/") || type.startsWith("video/") || this.previewMimetypes.includes(type)) ? this._selectedAttachment : preview;
+    this._submittedTextMode = false;
```
Comment (stetsche) [inline webcomponents/tool/src/main/frontend/packages/sakai-grader/src/SakaiGrader.js:336]: I am not sure what this does, can this always be false?

**Review by stetsche — COMMENTED** [review]
Hi @adrianfish! I was taking a look at this together with @st-manu. As it looks you also enabled the preview of html files, which is nice. Unfortunately we didn't manage to back-port this to 22 with your changes easily. I just have a minor comment related to your changes. I will approve this if it's not an issue.

File: webcomponents/tool/src/main/frontend/packages/sakai-grader/src/SakaiGrader.js (hunk 1 above)
Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/packages/sakai-grader/src/SakaiGrader.js:336]: It triggers the display of the student's submitted text, ie their submission from the text area.

Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/packages/sakai-grader/src/SakaiGrader.js:336]: I can help with the backport, if needed.

Comment (stetsche) [reply webcomponents/tool/src/main/frontend/packages/sakai-grader/src/SakaiGrader.js:336]: Thanks, there are no hurries with the back-port. We are applying the initial fix for now, with small adjustments.

**Review by stetsche — APPROVED** [review]
Thanks for the work @adrianfish!

**Pull request merged**